### PR TITLE
zh_trans:kubeadm_upgrade_apply.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.13 Features: set Milestone to 1.13 and Base Branch to dev-1.13
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+Upgrade your Kubernetes cluster to the specified version.
+
+```
+kubeadm upgrade apply [version]
+```
+
+### Options
+-->
+
+将Kubernetes集群升级到指定版本。
```
Comment (markthink) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md]: 将您的 Kubernetes 集群升级到指定版本

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+  </tbody>
+</table>
+-->
+
+  <tr>
+      <td colspan="2">--allow-experimental-upgrades</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">显示Kubernetes的不稳定版本作为升级替代方案，并允许升级到Kubernetes的alpha / beta / release候选版本。</td>
```
Comment (markthink) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md]: 中英文需要空格分隔
参考格式化：https://github.com/k8smeetup/k8s-official-translation

Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md]: release candidate version 中的 release candidate 是一个词组，不能分开。

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">帮助</td>
+    </tr>
+
+  <tr>
+      <td colspan="2">--ignore-preflight-errors stringSlice</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">一系列检查，其错误将显示为警告。示例：'IsPrivilegedUser，Swap'。值'all'忽略所有检查的错误。</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md]: 'IsPrivilegedUser，Swap' 中的逗号必须是半角。

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+    </tr>
+
+  <tr>
+      <td colspan="2">--image-pull-timeout duration&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 15m0s</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">等待control plane pod下载的最长时间。</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_apply.md]: control plane 控制面

**Pull request closed without merging**